frame/assets/src/weights.rs

```diff
@@ -128,12 +128,12 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 		// Minimum execution time: 17_565 nanoseconds.
 		Weight::from_parts(17_757_000, 5262)
 			// Standard Error: 15_192
-			.saturating_add(Weight::from_ref_time(13_799_167).saturating_mul(c.into()))
+			.saturating_add(Weight::from_parts(13_799_167, 0).saturating_mul(c.into()))
 			.saturating_add(T::DbWeight::get().reads(2_u64))
 			.saturating_add(T::DbWeight::get().reads((2_u64).saturating_mul(c.into())))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 			.saturating_add(T::DbWeight::get().writes((2_u64).saturating_mul(c.into())))
-			.saturating_add(Weight::from_proof_size(5180).saturating_mul(c.into()))
+			.saturating_add(Weight::from_parts(0, 5180).saturating_mul(c.into()))
```

Comment:
why do these two not just get combined into a single `from_parts`?

Reply:
The template is a bit inflexible right now, I can try but it will not look much better. Having these written as a term is very poor choice in retrospect.  
Something data-based would be better like https://github.com/paritytech/substrate/issues/12580#issuecomment-1304662025